tests/units/anta_tests/test_system.py

```diff
@@ -81,6 +81,25 @@
         "inputs": {"allowed_causes": ["ZTP"]},
         "expected": {"result": "success"},
     },
+    {
+        "name": "success-valid-reload-cause-fpga",
```

Comment:
ADd a failure test where the inputs is 
```
"inputs": {"allowed_causes": ["ZTP"]},
```

and the eos data is FPGA or User to validate this scenario